### PR TITLE
Packages e2e cluster spec

### PR DESCRIPTION
**Pull request by TerryHowe**
Very basic e2e test for packages cluster spec

### CODE REVIEW

**Comment by TerryHowe** [comment]
Borked presubmit
```
#7 ERROR: public.ecr.aws/h1r8a7l5/eks-anywhere-cluster-controller:latest: not found
```

File: test/framework/cluster.go [hunk 1]
```diff
@@ -1140,6 +1147,15 @@ func (e *ClusterE2ETest) SetPackageBundleActive() {
 	}
 }
 
+// ValidatingNoPackageController make sure there is no package controller.
+func (e *ClusterE2ETest) ValidatingNoPackageController() {
+	kubeconfig := e.kubeconfigFilePath()
+	_, err := e.KubectlClient.GetPackageBundleController(context.Background(), kubeconfig, e.ClusterName)
```
Comment (ivyostosh) [inline test/framework/cluster.go:1153]: Not sure how this fulfills the task to confirm there is no package controller. Will the existence of package controller always lead to an error state?

Comment (junshun) [reply test/framework/cluster.go:1153]: It checks if err == nil. Which means it was able to get a packagebundlecontroller.

Comment (TerryHowe) [reply test/framework/cluster.go:1153]: The helm chart creates a PBC

File: test/e2e/docker_test.go [hunk 2]
```diff
@@ -144,19 +144,19 @@ func TestDockerKubernetes122CuratedPackagesEmissarySimpleFlow(t *testing.T) {
...
 		framework.WithPackageConfig(t, packageBundleURI(v1alpha1.Kube122),
 			EksaPackageControllerHelmChartName, EksaPackageControllerHelmURI,
-			EksaPackageControllerHelmVersion, EksaPackageControllerHelmValues),
+			EksaPackageControllerHelmVersion, EksaPackageControllerHelmValues, nil),
 	)
 	runCuratedPackageEmissaryInstallSimpleFlow(test)
 }
 
-func TestDockerKubernetes123CuratedPackagesSimpleFlow(t *testing.T) {
+func TestDockerKubernetes125CuratedPackagesSimpleFlow(t *testing.T) {
```
[the remark this answers is not in the document]
Comment (junshun) [reply test/e2e/docker_test.go:152]: Lets make this a future issue

**Pull request merged**